jurion.js

```diff
@@ -0,0 +1,383 @@
+	var items = {};
+	var found = false;
+	var rows = ZU.xpath(doc, '//ul[contains(@class, "nvurlapi-result-list")]//h2/strong/a');
+	for (var i=0; i<rows.length; i++) {
+		var href = rows[i].href;
+		var title = ZU.trimInternal(rows[i].textContent);
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[href] = title;
+	}
+	return found ? items : false;
+}
+
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), function (items) {
+			if (!items) {
+				return true;
+			}
+			var articles = [];
+			for (var i in items) {
+				articles.push(i);
+			}
+			ZU.processDocuments(articles, scrape);
+		});
+	} else {
+		scrape(doc, url);
+	}
+}
+
+
+function scrape(doc, url) {
+	var type = detectWeb(doc, url);
+	var item = new Zotero.Item(type);
+	
+	item.title = ZU.xpathText(doc, '//div[@id="middle-column"]//h1[div[contains(@class, "addToList")]]');// [contains(@class, "artikel_titel") or contains(@class, "gesetz_ev_titel")]
+	var subtitle = ZU.xpathText(doc, '//div[contains(@class, "artikel_untertitel_thema")]');
+	if (subtitle) {
+		item.title += ' ' + subtitle;
+	}
+	
+	var author = getValueToLabel(doc, "Autor") || getValueToLabel(doc, "Verfasst von");
+	if (author) {
+		author = author.replace(/Dr\.?|Prof\.?|RA\.?/g, '');
+		item.creators.push(ZU.cleanAuthor(author, "author"));
+	}
+	
+	var journal = getValueToLabel(doc, "Zeitschrift");
+	if (journal) {
+		var pos = journal.indexOf('-');
+		if (pos>-1) {
+			item.publicationTitle = journal.substring(pos+1);
+			item.journalAbbreviation = journal.substring(0, pos);
+		} else {
+			item.publicationTitle = journal;
+		}
+	}
+	
+	var ref = getValueToLabel(doc, "Referenz");
+	if (ref) {
+		var m = ref.match(/(\d\d\d\d),\s+([\d\s\-IVX]*)\((.*)\)/);
+		if (m) {
+			//e.g. ZInsO 2010, 1959 - 1961 (Ausgabe 44 v. 28.10.2010)
+			// ZJJ 2010, 403 - 405 (Heft 4)
+			item.date = m[1];
+			item.pages = m[2].replace(/\s/g, '');
+			var parenthesis = m[3];
+			var d = parenthesis.match(/(\d\d?\.\d\d?\.\d\d\d\d)/);
+			if (d) {
+				item.date = ZU.strToISO(d[1]);
+				parenthesis = parenthesis.substring(0, d.index);
+			}
+			if (parenthesis.indexOf('Heft')>-1 || parenthesis.indexOf('Ausgabe')>-1) {
+				item.issue = parenthesis.replace(/\D/g, '');
+			}
+		} else {
+			//e.g. JurionRS 2012, 28843
+			var parts = ref.split(',');
+			if (parts.length == 2) {
+				item.pages = parts[1];
+				var space = parts[0].lastIndexOf(" ");
+				if (space>-1) {
+					if (type == "case") {
+						item.reporter = parts[0].substr(0, space);
+						item.reporterVolume = parts[0].substr(space+1);
+					} else if (type == "newspaperArticle") {
+						item.publicationTitle = parts[0].substr(0, space);
+					}
+				}
+			}
+		}
+	}
+	
+	//case
+	var date = getValueToLabel(doc, "Datum");
+	if (date) {
+		item.date = ZU.strToISO(date);
+	}
+	var court = getValueToLabel(doc, "Gericht");
+	if (court) {
```

Comment:
for all of thes -- why create a new variable and test for them? 
`item.court = getValueToLabel(doc, "Gericht");` is all you need. Just taking about court, docket Number, and form here. I understand you need the test for the others.